rllib/agents/a3c/a3c_torch_policy.py

```diff
@@ -38,14 +38,17 @@ def add_advantages(policy,
                    sample_batch,
                    other_agent_batches=None,
                    episode=None):
-    completed = sample_batch[SampleBatch.DONES][-1]
-    if completed:
-        last_r = 0.0
-    else:
-        last_r = policy._value(sample_batch[SampleBatch.NEXT_OBS][-1])
-    return compute_advantages(
-        sample_batch, last_r, policy.config["gamma"], policy.config["lambda"],
-        policy.config["use_gae"], policy.config["use_critic"])
+
+    with torch.no_grad():
```

Comment:
Could we actually move this into the postprocess method defined in `torch_policy_template`? That way it will work automatically for all torch policies and we don't need to clutter the individual ones.

Reply:
Sure, will do.